src/styles/blocks/section-title.scss

```diff
@@ -0,0 +1,27 @@
+.section-title {
+  margin: 0;
+  margin-bottom: 48px;
```

Comment:
Can't agree here. You have `row-gap` property to set spacing between these elements.